ansible/roles_ocp_workloads/ocp4_workload_amq_streams_dev_exp/tasks/post_workload.yml

```diff
@@ -0,0 +1,53 @@
+---
+# Implement your Post Workload deployment tasks here
+
+- name: Print no SSH
+  debug:
+    msg: "user.info: {{ item }}"
+  with_items:
+    - ""
+    - '**********'
+    - 'IMPORTANT: You cannot SSH into the shared cluster.'
+    - Please refer to the demo or lab instructions in order
+    - to get the steps on how to access the environment properly.
+    - '**********'
+    - ""
+
+- name: Get console route
+  command: oc get route -n openshift-console console -o json
+  register: routeconsole
+  ignore_errors: yes
+
+- name: Set webconsole address
+  when: routeconsole is succeeded
+  set_fact:
+    webconsole: "http://{{ routeconsole.stdout | from_json | json_query('spec.host') }}"
+
+- name: Print Overview
+  when: routeconsole is succeeded
+  debug:
+    msg: "{{ item }}"
+  with_items:
+    - "user.info: Openshift Master Console: {{ webconsole }}"
+
+- name: Get API for command line
+  command: oc whoami --show-server
+  register: showserver
+  ignore_errors: yes
```

Comment:
Can you use `k8s` module here ?
Same question regarding `ignore_errors`.

Reply:
This information doesn't seem to be exposed by the k8s modules.